src/js/modules/infragistics.ui.shared.js

```diff
@@ -720,7 +720,7 @@
 					this.handles.attr("disabled", "disabled");
 					this.element.addClass("ui-disabled");
 				} else {
-					this.handles.removeAttr("disabled");
+					this.handles[0].removeAttribute("disabled");
```

Comment:
Why is this needed if `attr()` is still supported?

Reply:
Apparently `removeAttr` should no longer be used when handling boolean attributes. It's being reported by jQuery migrate.

For tags that understand e.g. `disabled` (such as input), we could use removeProp which is the suggested way but in our code we freely use these on divs and such and removeProp actually sets them to false . (yeah it's a mess)